src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculable, Drawable {
+    static final int ARRAYS_LENGTH = 6;
+    static final int MAX_VALUE = 100;
```

Comment:
why did you put these constants here? Please also don't forget to add access modifier